### DISCUSSION TITLE
ns_show=False not working in version 2.0.0

### DISCUSSION THREAD

**Discussion by sebjoseph122** [elide-b/starbars#14] [General]
Hello, 

First of all thank you for the extremely useful package.

I just installed version 2.0.0 and am having trouble with ns_show. I don't want to show the ns bars, but they are showing even when set to false.

**Comment by elide-b** [comment]
Hey! This is a technical issue so I moved it to the issues and will follow up with you from there :)

Thank you for bringing this forward to me, I will work on it right now!
